identity-service/latest/tutorial/sso/saml.md

```diff
@@ -161,37 +161,7 @@ The properties listed that need to be set for Alfresco Content Services (ACS) ar
     |csrf.filter.referer | The referer value of ACS to prevent Cross Site Request Forgery (CSRF), for example `https://repo.example.com`|
     |csrf.filter.origin | The origin value of ACS to prevent Cross Site Request Forgery (CSRF), for example `https://repo.example.com/*`|
 
-2. Update the `share-config-custom.xml` file located by default in `$ALFRESCO_HOME/tomcat/shared/classes/alfresco/web-extension/`:
-
-    * Set the `CSRFPolicy` to true as in the following example:
```

Comment:
We need to remove only the second bullet point from this step - `* Add the following two rules to allow for signing out via SAML:`

Reply:
Fixed